.pipelines/templates/test-docs.yml

```diff
@@ -25,7 +25,8 @@ steps:
     
 - script: |
     set -eux  # fail on error    
-    pip install exdown 
+    pip install pytest-codeblocks 

```

Comment:
We can lock the version do avoid regressions

Reply:
done